doc/classes/AudioStreamPlayer2D.xml

```diff
@@ -64,6 +64,9 @@
 		<member name="max_polyphony" type="int" setter="set_max_polyphony" getter="get_max_polyphony" default="1">
 			The maximum number of sounds this node can play at the same time. Playing additional sounds after this value is reached will cut off the oldest sounds.
 		</member>
+		<member name="panning_strength" type="float" setter="set_panning_strength" getter="get_panning_strength" default="1.0">
+			Scales the panning strength for this node. Higher values will pan audio from left to right more dramatically than lower values.
```

Comment:
I think the relationship between this and the ProjectSettings value, and finally the actual panning that will be applied, is not very obvious from the docs. Maybe this would help:
```suggestion
			Scales the panning strength for this node by multiplying the base [member ProjectSettings.audio/general/2d_panning_strength] with this factor. Higher values will pan audio from left to right more dramatically than lower values.
```